src/interface/EigenLayer/IDelegationManager.sol

```diff
@@ -0,0 +1,188 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity >=0.5.0;
+
+import { IERC20 } from "openzeppelin/token/ERC20/IERC20.sol";
+import { IStrategy } from "./IStrategy.sol";
+
+interface IDelegationManager {
```

Comment:
why are we adding this instead of importing it from lib? to preserve the exact version?